tests/store/test_gcs_artifact_repo.py

```diff
@@ -33,34 +33,41 @@ def test_list_artifacts_empty(gcs_mock):
     assert repo.list_artifacts() == []
 
 
-def test_list_artifacts(gcs_mock):
-    dest_path = "/some/path/"
+def test_list_artifacts_with_subdir(gcs_mock):
+    dest_path = "/experiment_id/run_id/"
```

Comment:
Micronit: might be clearer to call this "artifact_root" or something similar

Reply:
Yep, I've renamed the variable